configure.ac

```
@@ -45,6 +45,15 @@ AS_IF([test "x$enable_analyzer" = "xyes"], [
 	EXTRA_CFLAGS="$EXTRA_CFLAGS -fanalyzer -Wno-analyzer-malloc-leak"
 ])
 
+AC_ARG_ENABLE([sanitizer],
+    AS_HELP_STRING([--enable-sanitizer=@<:@address | memory | undefined@:>@], [enable a compiler-based sanitizer (debug)]), [], [enable_sanitizer=no])
+AS_IF([test "x$enable_sanitizer" != "xno" ],
+	[AX_CHECK_COMPILE_FLAG([-fsanitize=$enable_sanitizer],[
```

```suggestion
	[AX_CHECK_COMPILE_FLAG([-fsanitize=$enable_sanitizer], [
```

Space nitpick.

Other than that, LGTM.